doc/cliet/README.md

```diff
@@ -0,0 +1,127 @@
+# jcli-client Document
+
+## How to get it
+
+go.mod
+```
+require github.com/jenkins-zh/jenkins-cli
+```
+
+## Configuration
+
+Examples of jcli configuration
+```
+- name: dev
+  url: http://192.168.1.10
+  username: admin
+  token: 11132c9ae4b20edbe56ac3e09cb5a3c8c2
+  proxy: http://192.168.10.10:47586
+  proxyAuth: username:password
+```
+So when you use jcl-client, you need to configure and read configuration files like this
+```
+type Config struct {
+	URL      string `yaml:"url"`
+	UserName string `yaml:"username"`
+	Token    string `yaml:"token"`
+}
+
+func GetJenkinsCore() (core client.JenkinsCore, e error) {
+	// 读取配置文件
```

Comment:
Please provide the comments in English as well. If you want to contribute a Chinese one, please create `README-zh.md`. Anyway, just using one language in a particular file.